tasks/add-fbc-contribution/add-fbc-contribution.yaml

```diff
@@ -135,8 +135,9 @@ spec:
             -p addArches=${add_arches} \
             -p hotfix=${hotfix} \
             -p stagedIndex=${staged_index} \
+            -p requestTimeoutSeconds=${request_timeout_seconds} \
```

Comment:
IIUC, this is just to make testing it possible. In `mocks.sh` we already echo all the options passed to the `mock_internal-request.txt` script. I think if this is just added for the test, it would be better to remove it and in the test check the `-t` value

Reply:
thanks for the pointer. updated test to use that pattern